### PR TITLE
docs - pxf max tomcat threads now user-configurable

### PR DESCRIPTION
**Pull request by lisakowen**
can now configure maximum tomcat threads in in $PXF_CONF/conv/pxf-env.sh.  add some more relevant intro content and update the procedure.

doc review site link:  http://docs-lisa-pxf1.cfapps.io/6-0/pxf/troubleshooting_pxf.html#pxf-threadcfg

### CODE REVIEW

**Comment by lisakowen** [comment]
does the PXF_MAX_THREADS value need to be in quotes?  i noticed that is was when i reviewed the code change PR.

**Comment by frankgh** [comment]
@lisakowen no, `PXF_MAX_THREADS` does not need to be in quotes

File: gpdb-doc/markdown/pxf/troubleshooting_pxf.html.md.erb [hunk 1]
```diff
@@ -263,37 +263,38 @@ Perform the following procedure to increase the heap size for the PXF agent runn
...
 
-The Tomcat default maximum number of threads is 300. Decrease the maximum number of threads to a value that works optimally for your Greenplum Database deployment. (If you plan to run large workloads on a large number of files in an external Hive data store, specify an even lower value.)
+The default maximum number of Tomcat threads for PXF is 200. The `PXF_MAX_THREADS` configuration property controls this setting.
 
-Perform the following procedure to decrease the maximum number of Tomcat threads for the PXF agent running on each segment host in your Greenplum Database deployment.
+PXF thread capacity is determined by the profile and whether or not the data is compressed. Set the maximum number of threads to a value that works optimally for your Greenplum Database deployment. If you plan to run large workloads on a large number of files in an external Hive data store, or you are reading compressed ORC or Parquet data, consider specifying a lower `PXF_MAX_THREADS` value.
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/troubleshooting_pxf.html.md.erb]: I think the second sentence here could be removed.

**Comment by lisakowen** [comment]
@frankgh - re calling them "configuration properties".  i have been using "environment variables" for things the user must explicitly set in their environment - PXF_CONF, PXF_HOST, PXF_PORT.  for those "things" that the user sets in the pxf config file pxf-env.sh, i have called them configuration properties because i wasn't sure if the user could actually explicitly set them outside of the pxf-env.sh file.  i.e. would the env setting or the file setting take precedence.  i can certainly change the terminology to "environment variable" if you think that is more accurate.

**Comment by frankgh** [comment]
@lisakowen that makes sense. I like `configuration properties`, but I wasn't sure why it is called that

**Comment by lisakowen** [comment]
merging soon, will address any further comments in a follow-on PR.

**Pull request merged**